### PR TITLE
chore(RHTAPWATCH-1002): Update button to point to konflux-ci

### PR DESCRIPTION
**Pull request by avi-biton**
Update button to point to konflux-ci

### CODE REVIEW

File: website/static/css/custom.css [hunk 1]
```diff
@@ -61,7 +61,7 @@
     height: 50%;
 }
 
-.doc-button {
+.try-button {
```
Comment (yftacherzog) [inline website/static/css/custom.css:64]: Maybe we can make the button less wide so we have smaller margins now that the text is shorter.

Comment (avi-biton) [reply website/static/css/custom.css:64]: Done

Comment (yftacherzog) [reply website/static/css/custom.css:64]: I think we can make the font size 18px or 20px.

Comment (avi-biton) [reply website/static/css/custom.css:64]: I think the 20px looks better.
Done

**Review by yftacherzog — APPROVED** [review]
lgtm

**Pull request merged**